<launch>
	<arg name="rviz" default="true" />

	<param name="save_pcd" type="bool" value="true" />
	<param name="voxel_size" type="double" value="1" />    <!-- For plane detection -->
	<param name="min_plane_size" type="int" value="20" />  <!-- For plane detection -->
	<param name="keyframe_size" type="int" value="3" />    <!-- Each keyframe is made of {keyframe_size} frames of input pcl -->
    <param name="window_size" type="int" value="75" />     <!-- Size of keyframes to be jointly optimized -->
	<param name="slide_size" type="int" value="20" />      <!-- Optimization will start with {slide_size} of new keyframes inputed -->

	<node pkg="balm2" type="run_real" name="balm" output="screen"/>

	<group if="$(arg rviz)">
		<node pkg="rviz" type="rviz" name="rviz" args="-d $(find balm2)/rviz_cfg/real.rviz"/>
	</group>  

</launch>